src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/websocket/WebSocketConstants.java

```diff
@@ -58,12 +57,16 @@ public class WebSocketConstants {
     public static final String CLIENT_ENDPOINT_ANNOTATION = "jakarta.websocket.ClientEndpoint";
 
     // Superclass
-    public static final String ENDPOINT_SUPERCLASS = "Endpoint";
     public static final String FQ_ENDPOINT_SUPERCLASS = "jakarta.websocket.Endpoint";
     public static final String IS_SUPERCLASS = "isSuperclass";
 
     public static final String[] WS_ANNOTATION_CLASS = { SERVER_ENDPOINT_ANNOTATION, CLIENT_ENDPOINT_ANNOTATION };
 
+    public static final String BOOLEAN = "java.lang.Boolean";
+    public static final String INTEGER = "java.lang.Integer";
+    public static final String LONG = "java.lang.Long";
+    public static final String DOUBLE = "java.lang.Double";
+    public static final String FLOAT = "java.lang.Float";
     public static final String STRING_CLASS_LONG = "java.lang.String";
     public static final String STRING_CLASS_SHORT = "String";
```

Comment:
I noticed that there are still several constants (such as `STRING_CLASS_SHORT` for unqualfied names that are still used by this class after your changes. I'm guessing they're being used somewhere but wondering whether you checked what is using them and why. If you haven't investigated this yet can you check? I suspect there might be another issue somewhere or at least obsolete code that we should clean up.

Reply:
I reviewed the usage of these names and identified that they were referenced [here](https://github.com/OpenLiberty/liberty-tools-intellij/blob/efe3fedbdff07739813239fb740b59b36a8efb75/src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/websocket/WebSocketDiagnosticsCollector.java#L391). According to [this commit](https://github.com/OpenLiberty/liberty-tools-intellij/commit/ca2f0c78fc21cc6e7769eb3d06bc5c2f72400a3c#diff-e1a14466e98b051537053ad0b6ba48a7e6e5fd0a2697f83cc3f70115ad461bf6L215-R189), `WebSocketConstants.SHORT_MESSAGE_CLASSES.contains(typeName)` was removed, leaving only `WebSocketConstants.LONG_MESSAGE_CLASSES.contains(typeName)` in use. Therefore, the switch case [here](https://github.com/OpenLiberty/liberty-tools-intellij/blob/efe3fedbdff07739813239fb740b59b36a8efb75/src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/websocket/WebSocketDiagnosticsCollector.java#L391) became unnecessary. I have removed it and refactored the code accordingly.